### ISSUE TITLE
Where do you define the loss functions present in the paper?

### ISSUE THREAD

**Issue by joshmyersdean** [lu-group/sbinn#3]
Thank you for the work! I do not see in the code where you define L_{ode} or L_{aux}, would you be able to point that out? Also do you have instructions on how to run the code?

Thank you!

**Comment by mitchelldaneker** [comment]
When running sbinn_tf.py or sbinn_pytorch.py, the L_{ode} and L_{aux} functions are defined through the use of DeepXDE. The ode loss is given via the `ODE(t, y)` function and implemented when creating the `dde.data.PDE `instance. Aux loss is essentially part of the data loss, and is another supervised loss defined and computed by DeepXDE. For more information on how these losses are calculated, see the [GitHub](https://github.com/lululxvi/deepxde) for DeepXDE.

Instructions for installing DeepXDE can be found on the DeepXDE [GitHub](https://github.com/lululxvi/deepxde). The Julia code for structural identifiability analysis can run as is ([structural_identifiability.ipynb](https://github.com/lu-group/sbinn/blob/main/structural_identifiability.ipynb)). To run the Python code, you will need to first run [data_generation.py](https://github.com/lu-group/sbinn/blob/main/sbinn/data_generation.py), which will create the data and save it. Then you can run [sbinn_tf.py](https://github.com/lu-group/sbinn/blob/main/sbinn/sbinn_tf.py) if you are using one of the two tensorflow backends or [sbinn_pytorch.py](https://github.com/lu-group/sbinn/blob/main/sbinn/sbinn_pytorch.py) if using the pytorch backend of DeepXDE. Make sure to have [variable_to_parameter_transform.py](https://github.com/lu-group/sbinn/blob/main/sbinn/variable_to_parameter_transform.py) downloaded so it can convert the variables into their true values. The result of running these will provide inferred variables in a .csv file. Practical identifiability analysis can run as is in Julia or can be changed to fit the parameters you have inferred ([practical_identifiability.jl](https://github.com/lu-group/sbinn/blob/main/practical_identifiability.jl)).

**Comment by joshmyersdean** [comment]
Thank you so much!
